tools/run_tests/python_utils/check_on_pr.py

```diff
@@ -118,3 +119,12 @@ def check_on_pr(name, summary, success=True):
         })
     print('Result of Creating/Updating Check on PR:',
           json.dumps(resp.json(), indent=2))
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print('Please specify the name of the check item.')
```

Comment:
Can we have a slightly more descriptive "usage" message? I'm also not 100% clear on what a "check item" is.

Reply:
Hahaha. My bad, I guess the ideal way is to introduce the `ArgParser`.